doc/Language/functions.rakudoc

```diff
@@ -1018,6 +1018,55 @@ dispatch, or it could be the inner routine from a C<wrap>.
 Fortunately, we have a series of re-dispatching tools that help us to make
 it easy.
 
+=head2 The Next Tools
+
+The table below represents the next tools in relation to three attributes:
+
+=item1 B<Returns:> "Yes" means that it returns, and the code will continue from
+that point.  "No" means that the it never returns to this function -- it's as 
+though the call has a built-in "return".  
+
+=item1 B<Parameters:> Specifically, whether the parameters from the current
+function are reused in the call, or whether you need to specify some 
+parameters.  In the case of "nextcallee", there are no parameters needed, 
+because they're instead passed to the return value when it's called.  
+
+=item1 B<Referee:> This is the method that will be called by the tool (the 
+reciprocal of "referer".  It can be one of:
+
+=item2 B<next:> The next item in the chain
+
+=item2 B<self:> It will call itself again, as in recursion
+
+=item2 B<preset:> It will call the item that was saved by "nextcallee"
+
+=begin table
+
+                    | Returns | Parameters | Referee*
+    ================+=========+============+========
+    nextsame        | No      | Reuse      | next
+    nextwith        | No      | Specify    | next
+    callsame        | Yes     | Reuse      | next
+    callwith        | Yes     | Specify    | next
+    nextcallee      | Yes     | None       | next
+    samewith        | Yes     | Specify    | self
+    nextcallee's rv | Yes     | Specify    | preset
```

Comment:
What's the rv?

Reply:
Return Value.  I've added a comment below the table that will hopefully explain.